src/EditModel/event-program/assign-data-elements/AssignDataElements.js

```diff
@@ -224,12 +236,28 @@ function AssignDataElements(props, { d2 }) {
     const itemStore = Store.create();
     const assignedItemStore = Store.create();
 
+    //Fix for DHIS2-4369 where some program stages may contain other dataelements than TRACKER
+    //This is due to a database inconsistency. This fix makes it possible to show and be able to remove these
+    //elements from the UI
+    const otherElems = props.model.programStageDataElements
+    .filter(v => {
```

Comment:
You could destructure the function parameters instead to remove a line:
`.filter({ dataElement: domainType }) => {`